packages/trader/src/Modules/Trading/Components/Elements/PayoutPerPoint/payout-per-point-mobile-input.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Icon, PageOverlay, Popover, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import WheelPickerMobile from '../../Form/WheelPicker/wheel-picker-mobile';
+import './payout-per-point.scss';
+import Fieldset from 'App/Components/Form/fieldset';
+import { LabelPairedChevronsDownCaptionRegularIcon, LabelPairedChevronsUpCaptionRegularIcon } from '@deriv/quill-icons';
+
+const PayoutPerPointMobileInput = ({
+    togglePayoutWheelPicker,
+    payoutChoices,
+    onPayoutClick,
+    selectedBarrier,
+    payout_per_point,
+    currency,
+}: {
+    togglePayoutWheelPicker: () => void;
+    payoutChoices: number[];
+    selectedBarrier: string;
+    onPayoutClick: (val: number) => void;
+    currency: string;
+    payout_per_point?: number;
+}) => {
+    const [initialPayout, setInitialPayout] = useState<number | null>(null);
+
+    const [payoutValue, setPayoutValue] = useState(payoutChoices[2] || payoutChoices[0]);
+
+    useEffect(() => {
+        if (initialPayout == null && payout_per_point) {
+            setInitialPayout(payout_per_point);
+        }
+    }, []);
```

Comment:
yeah actually we saving the initial value of the payout_per_point . and it'll be same and won't have any dependency. 
so that if the picker overlay is closed we can again assign this initial value to the payout_per_point